usecases/modules/module_config_init_and_validate.go

```diff
@@ -35,7 +35,7 @@ func (p *Provider) SetClassDefaults(class *models.Class) {
 		return
 	}
 
-	cfg := NewClassBasedModuleConfig(class, class.Vectorizer)
+	cfg := NewClassBasedModuleConfig(class, class.Vectorizer, "")
```

Comment:
shouldn't be here instead of `""` a call to class `class.Tenant`?

Reply:
oh, I see now, that class has no `tenant` in it

Reply:
I'm wondering if here we need tenant, can you create a follow up issue on me checking this later? for now we can leave it as it is